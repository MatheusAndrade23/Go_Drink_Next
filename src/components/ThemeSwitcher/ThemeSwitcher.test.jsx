import { screen } from '@testing-library/react';
import { renderTheme } from '../../tests/render-base.js';
import { ThemeSwitcher } from '.';

describe('<ThemeSwitcher />', () => {
  it('should render', () => {
    const { container } = renderTheme(<ThemeSwitcher />);
    expect(container).toMatchSnapshot();
  });
});
