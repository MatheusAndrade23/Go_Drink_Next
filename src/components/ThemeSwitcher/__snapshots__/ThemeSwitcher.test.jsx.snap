// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThemeSwitcher /> should render 1`] = `
.c0 {
  position: fixed;
  background-color: #EEEEEE;
  color: #222831;
  bottom: 10px;
  right: 20px;
  z-index: 6;
  opacity: 0.5;
  cursor: pointer;
  font-weight: bold;
  height: 30px;
  width: 200px;
  border: none;
  -webkit-transition: 0.5s;
  transition: 0.5s;
}

<div>
  <button
    class="c0"
    title="lightTheme"
  >
    lightTheme
  </button>
</div>
`;
